src/FamilyHubs.ServiceDirectory.Core/Cookies/RawCookieExtensions.cs

```diff
@@ -45,7 +45,7 @@ public static void AppendRawCookie(this HttpResponse response, string key, strin
             cookieValue.Append("; HttpOnly");
         }
 
-        if (options.SameSite != SameSiteMode.Unspecified)
+        if (options.SameSite != SameSiteMode.None)
```

Comment:
I think this change needs to be reverted.

`SameSiteMode.Unspecified` is a MS only value to basically say 'don't set `SameSite` in the cookie'.

```
    /// <summary>No SameSite field will be set, the client should follow its default cookie policy.</summary>
```

Whereas `None` is a [real value accepted for the SameSite option](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite), so should be set in the header if `options.SameSite` is set to `None`.

Reply:
moved to web project and removed wrong package reference